src/main/java/com/rtb/projectmanagementtool/project/AddUserToProjectServlet.java

```diff
@@ -0,0 +1,56 @@
+/** Servlet responsible for adding users to a project */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/add-user-to-project")
+public class AddUserToProjectServlet extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+
+    // initialize controllers
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    ProjectController projectController = new ProjectController(datastore);
+    UserController userController = new UserController(datastore);
+
+    // get request parameters
+    Long projectId = Long.parseLong(request.getParameter("projectId"));
+    String userName = request.getParameter("user-name");
+    String userRole = request.getParameter("user-role");
+
+    // create objects
+    ProjectData project = projectController.getProjectById(projectId);
+    UserData user = userController.getUserByName(userName);
+
+    // if user is not in database or they are already in the project,
```

Comment:
Probably should let the user know about this.  We don't want them thinking they added someone but the system actually didn't